vit-servicing-station-lib/src/db/models/snapshot.rs

```diff
@@ -0,0 +1,165 @@
+use crate::db::{
+    schema::{contributors, snapshots, voters},
+    Db,
+};
+use diesel::{ExpressionMethods, Insertable, Queryable};
+use jormungandr_lib::crypto::account::Identifier;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub struct Snapshot {
+    /// Tag - a unique identifier of the current snapshot
+    pub tag: String,
+    /// Timestamp for the latest update of the current snapshot
+    #[serde(alias = "lastUpdated")]
+    #[serde(deserialize_with = "crate::utils::serde::deserialize_unix_timestamp_from_rfc3339")]
+    #[serde(serialize_with = "crate::utils::serde::serialize_unix_timestamp_as_rfc3339")]
+    pub last_updated: i64,
+}
+
+impl Queryable<snapshots::SqlType, Db> for Snapshot {
```

Comment:
Is there a reason we're not deriving this?

Reply:
Good catch ! Updated it !